int32 axis6
int32 axis7
int32 button0
int32 button1
int32 button2
